CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
 cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 project(MAPBOX_BASE LANGUAGES CXX C)
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+option(WITH_CXX11ABI "Use the C++11 ABI" ON)
```

Comment:
I don't think this is needed, because the package is not pre-built.

Reply:
We're compiling `mapbox-base-io`, which now has a C++ definition file.